src/components/ButtonComponent/ButtonComponent.css

```
@@ -0,0 +1,15 @@
+.RoundAddButton {
+ width: 56px;
+ height: 56px;
+ background-color: #F7B21F;
+ border-radius: 50%;
+ border: none;
+ box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+
+.RoundAddButton:focus {
```

The button should remove the shadow hover. As of now its not doing anything
```suggestion
.RoundAddButton:hover {
```

---

Hey @Mubangizi Allan, this is not a mistake.
The effect occurs on click, not on hover.